float32[] dlib_X
float32[] dlib_Y
int32[] dlib_face_index
